ui/src/components/References/NoteReference.tsx

```diff
@@ -64,7 +64,7 @@ function NoteReference({
   const prettyDate = makePrettyDate(new Date(outline.sent));
 
   return (
-    <div className="note-inline-block not-prose group max-w-[600px]">
```

Comment:
there was some reason I did this that I'm having trouble remembering, but I think it had to do with nested refs in notes. will look real quick

Reply:
Cool, just added it, looks good.